### ISSUE TITLE
Feature Request: Hide Project Title on Cards

### ISSUE THREAD

**Issue by Uduse** [ZenHubIO/support#1171]
I feel like project titles on cards have decent visual impact yet offers no real information (you almost always know what board you are on, and even if you don't, it's right on top of your screen).

Therefore, if it's possible, I think removing project title on cards might be a good idea. Any consideration so far to do so?

**Comment by ripla** [comment]
Agreed, it's way too noisy. I like to keep all the other info, and set`.zhc-issue-card__heading__main { display: none;}`. Looks so much nicer.
